src/Parsers/ASTExplainQuery.h

```diff
@@ -104,6 +108,11 @@ class ASTExplainQuery : public ASTQueryWithOutput
     ASTPtr table_function;
     ASTPtr table_override;
 
+    /// To convert `SELECT * FROM (EXPLAIN PIPELINE header = 1 SELECT 123)` to `SELECT * FROM viewExplain("EXPLAIN PIPELINE", "header = 1", SELECT 123)`
```

Comment:
If ExplainKind is public, it make sense to make `toString`, `fromString` public as well.